splunk-quarkus/src/test/java/com/redhat/console/integrations/splunk/SplunkUrlCleanerTest.java

```diff
@@ -0,0 +1,50 @@
+package com.redhat.console.integrations.splunk;
+
+import io.quarkus.test.junit.QuarkusTest;
+import org.apache.camel.Exchange;
+import org.apache.camel.quarkus.test.CamelQuarkusTestSupport;
+import org.junit.jupiter.api.Test;
+
+import static com.redhat.console.integrations.splunk.SplunkUrlCleaner.SERVICES_COLLECTOR;
+import static com.redhat.console.integrations.splunk.SplunkUrlCleaner.SERVICES_COLLECTOR_EVENT;
+import static com.redhat.console.integrations.splunk.SplunkUrlCleaner.TARGET_URL;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+@QuarkusTest
+public class SplunkUrlCleanerTest extends CamelQuarkusTestSupport {
+
+    private static final String VALID_TARGET_URL = "https://i.am.valid.com";
+
+    @Test
+    void shouldNotChangeUrlWhenPathIsOk() throws Exception {
+
+        Exchange exchange = createExchangeWithBody("I am not used in this test!");
+        exchange.setProperty(TARGET_URL, VALID_TARGET_URL);
+
+        new SplunkUrlCleaner().process(exchange);
+
+        assertEquals(VALID_TARGET_URL, exchange.getProperty(TARGET_URL, String.class));
+    }
+
+    @Test
+    void shouldRemoveServicesCollectorPath() throws Exception {
+
+        Exchange exchange = createExchangeWithBody("I am not used in this test");
+        exchange.setProperty(TARGET_URL, VALID_TARGET_URL + SERVICES_COLLECTOR);
```

Comment:
I wouldn't use imported constants that are used within the unit being tested.

Reply:
There are 3 constants in this line. Which one made you write that comment?

`TARGET_URL` and `SERVICES_COLLECTOR` come from the runtime code and make the code maintenance quicker and less error prone.

Reply:
`SERVICES_COLLECTOR` (and `SERVICES_COLLECTOR_EVENT`)